pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[project]
+name = "head-orientation-node-for-comfyui---by-pablogfx"
```

Comment:
Maybe rename it to something simpler? `head-orientation-node`? 
Don't need to say ComfyUI and your publisher name will include pablogfx